implementation/src/main/java/io/smallrye/openapi/runtime/scanner/OpenApiAnnotationScanner.java

```diff
@@ -128,6 +131,8 @@
  */
 public class OpenApiAnnotationScanner {
 
+    private static final DotName COMPLETION_STAGE_NAME = DotName.createSimple(CompletionStage.class.getName());
```

Comment:
Maybe put this in https://github.com/smallrye/smallrye-open-api/blob/master/implementation/src/main/java/io/smallrye/openapi/api/OpenApiConstants.java

Reply:
Done